bundles/org.smarthomej.binding.viessmann/README.md

```diff
@@ -1,15 +1,19 @@
 # Viessmann Binding
 
+<img src="/bundles/org.smarthomej.binding.viessmann/doc/viessmann_wordmark_rgb_1_vitorange.png" width="140"/>
```

Comment:
I doubt that works on the documentation website. I would suggest to change it to `org.smarthomej.binding.viessmann/doc/viessmann_wordmark_rgb_1_vitorange.png`, then I can try to add that to the website.

Reply:
I changed it, but now it no longer works on github. Let's see what happens on the doc website.